marlowe-runtime/sync-api/Language/Marlowe/Protocol/Query/Types.hs

```diff
@@ -64,7 +68,21 @@ instance Monoid ContractFilter where
     , roleCurrencies = mempty
     }
 
+data RuntimeStatus = RuntimeStatus
```

Comment:
My comment on the DTO applies here also : 

``````haskell
data RuntimeStatus 
  = RuntimeStatus
     {  node :: NodeStatus 
     ,  chainIndexer :: ChainIndexerStatus
     ,  tip :: ChainTip
     ,  version :: Version } 

data ChainIndexerStatus 
   = ChainIndexerStatus  
      { tip :: ChainTip
      , version :: Version 
      } 

data NodeStatus 
     = NodeStatus 
        { tip :: ChainTip
        , networkId :: NetworkId  
        , version :: Version -- possible ?
        } 
```

Reply:
See my comment above